### PR TITLE
Removed bombs, replaced with stopwatch

### PR DESCRIPTION
**Pull request by cecileboucheron**
We're getting rid of the bomb icon and replacing it with a stopwatch everywhere. 
@keybase/react-hackers 

image

image

image

### CODE REVIEW

File: shared/chat/conversation/messages/wrapper/exploding-meta/index.js [hunk 1]
```diff
@@ -115,10 +115,10 @@ class ExplodingMeta extends React.Component<Props, State> {
     let children
     switch (this.state.mode) {
       case 'countdown':
-        let bombIconSize = isMobile ? 22 : 16
+        let timerIconSize = isMobile ? 22 : 16
         if (isAndroid) {
           // icon is 24 high and clips edge of container on android. workaround
```
Comment (adamjspooner) [inline shared/chat/conversation/messages/wrapper/exploding-meta/index.js]: Do you know if this is still needed?

Comment (cecileboucheron) [reply shared/chat/conversation/messages/wrapper/exploding-meta/index.js]: @chrisnojima Do you know if this is still needed?

Comment (adamjspooner) [reply shared/chat/conversation/messages/wrapper/exploding-meta/index.js]: ...taking a look.

Comment (chrisnojima) [reply shared/chat/conversation/messages/wrapper/exploding-meta/index.js]: lets touch this stuff very little, i'm in the middle of all of this

**Pull request merged**